jsonrpc/eth_endpoint.go

```diff
@@ -81,6 +83,9 @@ func (e *Eth) SendRawTransaction(input string) (interface{}, error) {
 
 // SendTransaction creates new message call transaction or a contract creation, if the data field contains code.
 func (e *Eth) SendTransaction(arg *txnArgs) (interface{}, error) {
+	if !e.d.isAccountAvailable(*arg.From) {
+		return nil, errors.New("unknown account")
```

Comment:
Confirming GETH and possibly other clients have this logic as well, as it seemed fishy to me when I first looked at it 👍

Reply:
This is not required, since we don't support account manager to sign transactions. Fixed it [Commit-link](https://github.com/0xPolygon/polygon-sdk/commit/7ed9279cd30c72299f4073dc9bf906cde80e4724)